user/pages/01.modular/_news/newslist.en.md

```diff
@@ -1,4 +1,5 @@
 ---
+published: false
```

Comment:
Lucia is adding news today, so let's leave the section visible (same for .sk.md)